llpc/test/shaderdb/OpAtomicAnd_TestInt64ImageAtomicAnd.spvas

```diff
@@ -0,0 +1,61 @@
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL: {{^// LLPC}} SPIRV-to-LLVM translation results
+; SHADERTEST-LABEL: {{^// LLPC}} SPIR-V lowering results
+; SHADERTEST: call i64 (...) @llpc.call.image.atomic.i64(i32 8
+; SHADERTEST-LABEL: {{^// LLPC}} pipeline patching results
+; SHADERTEST: call i64 @llvm.amdgcn.image.atomic.and.2d.i64.i32
+; SHADERTEST: AMDLLPC SUCCESS
+
```

Comment:
Do you have GLSL source code? if have, please add it a comment. then, we can replace SPV asm with glsl source once glslang support this new extension.

Reply:
The GLSL extension is under way. So I can only use SPIR-V assembly for this testing. I will replace them with GLSL once it is ratified.